### ISSUE TITLE
Which ones are the support and resistance levels?

### ISSUE THREAD

**Issue by vmorris1959** [finnhubio/Finnhub-API#388]
In the response for https://finnhub.io/api/v1/scan/support-resistance I get an array of levels but:

1. Will the response always contain 6 values?
2. What values correspond to resistance and support? Is it 3 for each?
3. Is there any for pivot?

Sample response:

`
{
  "levels": [
    1.092360019683838,
    1.1026300191879272,
    1.113450050354004,
    1.1233500242233276,
    1.134719967842102,
    1.1513700485229492
  ]
}
`

**Comment by boherna** [comment]
I agree is confusing, the response should be a dictionary with a label for each value, e.g., s1, s2, s3, r1, r2, r3, etc.

**Comment by finnhubio** [comment]
Hi there,

Our Machine learning algorithm currently detects clusters of price and output those levels. Whether a level is support or resistance is simply determined by whether the price is currently above or below such level. Once a support is broken, it becomes a resistance and vice versa so we decided not to label those levels specifically.

Hope it helps. Let us know if you have any further questions
